epoxy-adapter/src/main/java/com/airbnb/epoxy/ModelGroupHolder.kt

```diff
@@ -58,40 +58,31 @@ class ModelGroupHolder : EpoxyHolder() {
     }
 
     private fun createViewStubData(viewGroup: ViewGroup): List<ViewStubData> {
-        val stubs = ArrayList<ViewStubData>(4)
+        return ArrayList<ViewStubData>(4).apply {
```

Comment:
Just curious, why don't you use Kotlin list : `mutableListOf<ViewStubData>()` ?

Reply:
I believe it was so I could give an initial capacity value

Reply:
Ok, why 4 ?

Reply:
it's a somewhat arbitrary guess - groups are generally small, but non zero size